blessings/__init__.py

```diff
@@ -0,0 +1,560 @@
+"""A thin, practical wrapper around terminal coloring, styling, and
+positioning"""
+
+from contextlib import contextmanager
+import curses
+from curses import setupterm, tigetnum, tigetstr, tparm
+from fcntl import ioctl
+
+try:
+    from io import UnsupportedOperation as IOUnsupportedOperation
+except ImportError:
+    class IOUnsupportedOperation(Exception):
+        """A dummy exception to take the place of Python 3's
+        ``io.UnsupportedOperation`` in Python 2"""
+
+from os import isatty, environ
+from platform import python_version_tuple
+import struct
+import sys
+from termios import TIOCGWINSZ
+
```

Comment:
Oops, I have Blessings installed globally on my machine and didn't think to see if it was importing other 3rd party libraries.

I thought it was self-contained. If not I'll remove it, I'm just using it for flourish and could do it manually if I really wanted.